tests/pr_test.go

```diff
@@ -196,6 +197,7 @@ func TestAgentsSolutionInSchematics(t *testing.T) {
 		options.TerraformVars = []testschematic.TestSchematicTerraformVar{
 			{Name: "ibmcloud_api_key", Value: options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"], DataType: "string", Secure: true},
 			{Name: "cloud_monitoring_instance_region", Value: region, DataType: "string"},
+			{Name: "provider_visibility", Value: "public", DataType: "string"},
```

Comment:
Since we are running the test in schematics so please remove this as it will take the default input from variables.tf.